certify-mosipid-identity.properties

```diff
@@ -25,6 +29,9 @@ mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 
 ## ------------------------------------------- Use specific default overriding properties ------------------------------------------------------------
 
+mosip.certify.domain.url=https://${mosip.injicertify.host}
```

Comment:
I think the value should be different here like https://${mosip.injicertify.mosipid.host}

Reply:
changed specific to three namespaces